virt/virtual_machines/virtual_disks/virt-using-container-disks-with-vms.adoc

```diff
@@ -23,6 +23,8 @@ If your container registry does not have TLS you must add it as an insecure regi
 
 include::modules/virt-disabling-tls-for-registry.adoc[leveloffset=+1]
 
+include::modules/virt-golden-images.adoc[leveloffset=+1]
```

Comment:
I am not sure it is a good idea to put a concept module at the end of the assembly. That pretty much guarantees it will not be read. I think it should be placed above "about container disks". Then rewrite the introduction so that it mentions golden images.